examples/jetstream/ack-ack/rust/main.rs

```diff
@@ -0,0 +1,60 @@
+use std::{env};
+use async_nats::jetstream;
+use futures::StreamExt;
+
+#[tokio::main]
+async fn main() -> Result<(), async_nats::Error> {
+    let nats_url = env::var("NATS_URL").unwrap_or_else(|_| "nats://localhost:4222".to_string());
+
+    let client = async_nats::connect(nats_url).await?;
+    let jetstream = jetstream::new(client);
+
+    let stream = jetstream
+        .create_stream(jetstream::stream::Config {
+            name: "EVENTS".to_string(),
+            subjects: vec!["event.foo".to_string()],
+            ..Default::default()
+        })
+        .await?;
+
+    let _ = jetstream.publish("event.foo", "1".into()).await;
+    let _ = jetstream.publish("event.foo", "2".into()).await;
+
+    let mut consumer = stream
+        .create_consumer(async_nats::jetstream::consumer::pull::Config { ..Default::default()})
+        .await?;
+
+    let ci = consumer.info().await?;
+    println!("Consumer 1");
+    println!("  Start\n    # pending messages: {}\n    # messages with ack pending: {}", ci.num_pending, ci.num_ack_pending);
+
+    let message = consumer.fetch().max_messages(1).messages().await?.next().await.unwrap()?;
```

Comment:
I would not unwrap the optional without comment here.
Let's explain to the user what is happening here:  that `messages()` returns an iterator that is closed when `None` is returned.

maybe this is giving the user a better idea what is happening?
```rust
let fetch = consumer.fetch().max_messages(1).messages().await?;
let message = match fetch.next().await.expect("should get one message");
```